--- install/lib/ruby/gems/1.9.1/gems/passenger-3.0.18/bin/passenger-install-nginx-module.orig	2012-11-24 17:36:20.728534653 -0500
+++ install/lib/ruby/gems/1.9.1/gems/passenger-3.0.18/bin/passenger-install-nginx-module	2012-11-24 17:41:22.060518427 -0500
@@ -412,7 +412,7 @@
 				end
 				Dir.chdir('/tmp') do
 					# Nginx checks for PCRE in multiple places...
-					system("(gcc -I/usr/local/include -I/usr/include/pcre " <<
+                                        system("(gcc #{ENV['CFLAGS']} -I/usr/local/include -I/usr/include/pcre " <<
 						"-I/usr/pkg/include -I/opt/local/include " <<
 						"-c passenger-check.c) >/dev/null 2>/dev/null")
 				end
